src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -181,7 +181,7 @@ public static class StandardTrainersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear classification model trained with <see cref="SdcaCalibratedBinaryTrainer"/>.
+        /// Predict a target using a linear classification model trained with <see cref="SdcaLogisticRegressionBinaryTrainer"/>.
```

Comment:
>SdcaLogisticRegressionBinaryTrainer [](start = 89, length = 35)

Similar to `SymSgd`, now `Sdca` has `LogisticRegression` and `Binary`.

Reply:
Yes, adding both of `LogisticRegression` and `Binary` to trainer classes was our decision. For API, we only append `LogisticRegression` because `Binary` naturally comes from their context.

---
In reply to: [268859905](https://github.com/dotnet/machinelearning/pull/3034#discussion_r268859905) [](ancestors = 268859905)